internal/server/models/organization.go

```diff
@@ -21,6 +25,12 @@ func (o *Organization) ToAPI() *api.Organization {
 	}
 }
 
+// TODO(orgs): remove
+func (o *Organization) BeforeCreate(_ *gorm.DB) error {
+	// allow an org with id=0
+	return nil
+}
```

Comment:
To get around all the test failures when we create the default org, the last commit in this PR allows creating the default org with ID=0.

I'm tracking this cleanup in #2903, and all of the changes are done in a small commit, so it's going to be easy to revert when the tests are ready for it.